pkg/distro/defs/rhel-7/distro.yaml

```diff
@@ -134,15 +255,265 @@ image_config:
+    payload_pipelines: ["os", "image", "vpc", "xz"]
+    exports: ["xz"]
+    compression: "xz"
+    bootable: true
+    # RHEL 7 qemu vpc subformat does not support force_size
+    disk_image_vpc_force_size: false
+    platforms:
+      - <<: *x86_64_uefi_vendor_platform
+        image_format: "vhd"
+    image_config:
+      timezone: "Etc/UTC"
+      locale: "en_US.UTF-8"
+      gpgkey_files:
+        - "/etc/pki/rpm-gpg/RPM-GPG-KEY-microsoft-azure-release"
+        - "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+      selinux_force_relabel: true
+      authconfig: {}
+      update_default_kernel: true
+      default_kernel: "kernel-core"
+      sysconfig:
+        networking: true
+        no_zero_conf: true
+      enabled_services:
+        - "cloud-config"
+        - "cloud-final"
+        - "cloud-init-local"
+        - "cloud-init"
+        - "firewalld"
+        - "NetworkManager"
+        - "sshd"
+        - "waagent"
+      sshd_config:
+        config:
+          ClientAliveInterval: 180
+      modprobe:
+        - filename: "blacklist-amdgpu.conf"
+          commands:
+            - command: blacklist
+              modulename: "amdgpu"
+        - filename: "blacklist-intel-cstate.conf"
+          commands:
+            - command: blacklist
+              modulename: "intel_cstate"
+        - filename: "blacklist-floppy.conf"
+          commands:
+            - command: blacklist
+              modulename: "floppy"
+        - filename: "blacklist-nouveau.conf"
+          commands:
+            - command: blacklist
+              modulename: "nouveau"
+            - command: blacklist
+              modulename: "lbm-nouveau"
+        - filename: "blacklist-skylake-edac.conf"
+          commands:
+            - command: blacklist
+              modulename: "skx_edac"
+      cloud_init:
+        - filename: "06_logging_override.cfg"
+          config:
+            output:
+              all: "| tee -a /var/log/cloud-init-output.log"
+        - filename: "10-azure-kvp.cfg"
+          config:
+            reporting:
+              logging:
+                type: "log"
+              telemetry:
+                type: "hyperv"
+        - filename: "91-azure_datasource.cfg"
+          config:
+            datasource:
+              azure:
+                apply_network_config: false
+            datasource_list:
+              - "Azure"
+      pwquality:
+        config:
+          minlen: 6
+          minclass: 3
+          dcredit: 0
+          ucredit: 0
+          lcredit: 0
+          ocredit: 0
+      waagent_config:
+        config:
+          "ResourceDisk.Format": false
+          "ResourceDisk.EnableSwap": false
+      rhsm_config:
+        "no-subscription":
+          yum_plugin:
+            subscription_manager:
+              enabled: false
+          subman:
+            rhsmcertd:
+              auto_registration: true
+            rhsm:
+              manage_repos: false
+        "with-subscription":
```

Comment:
I think we do not generate manifests for this, i.e. I think this may not be tested